frontend/src/metabase-types/api/setup.ts

```diff
@@ -12,3 +12,9 @@ export interface SetupCheckListTask {
   triggered: boolean;
   is_next_step: boolean;
 }
+
+export type UsageReason =
```

Comment:
Technically in the future this will be sent to the BE, should it be there when that will happen?

Reply:
It's hard to say. Most types under `/api` are used for both the request and the response. In our case, this would only be for the request if I'm not mistaken. I'd say it's fine to stay here then. Because it'd be weird for the request type to stay somewhere else.